### DISCUSSION TITLE
How do I manually end phase 4?

### DISCUSSION THREAD

**Discussion by chialoser** [Chia-Network/chia-blockchain#4615] [Support]
The video card fell off the computer, I had to reboot the computer and 6 rafts did not end in phase 4, how does the 4th phase take place? Maybe I can manually or write a script and finish these rafts.

Starting phase 4/4: Write Checkpoint tables into "C: \\ tmp \\ plot-k32 - *. Plot.2.tmp" ... Mon May 10 01:52:02 2021

Starting to write C1 and C3 tables

Bucket 0 uniform sort. Ram: 3.826GiB, u_sort min: 0.563GiB, qs min: 0.281GiB.

...

**Comment by Plopsi** [comment]
A '.plot.tmp' file with the right size you could be fine you can try to use chia plot check to check if it's fine after renaming it to .plot but all files which have not finished are lost - you can delete the tmp-files to free disk-space.

**Comment by chialoser** [comment]
> A '.plot.tmp' file with the right size you could be fine you can try to use chia plot check to check if it's fine after renaming it to .plot but all files which have not finished are lost - you can delete the tmp-files to free disk-space.

from chiapos import DiskPlotter
plotter.create_plot_disk(
https://github.com/Chia-Network/chia-blockchain/blob/main/chia/plotting/create_plots.py

DiskPlotter
https://github.com/Chia-Network/chiapos/blob/main/python-bindings/chiapos.cpp

In order to finish the rafts, I only need tmp files, right? Judging by the source
https://github.com/Chia-Network/chiapos/blob/main/src/phase4.hpp

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
